modules/developers-guide/pages/tutorials/phonebook.adoc

```diff
@@ -170,28 +166,28 @@ Note that, in this case, the command returns `+(null)+` because the phone number
 dfx canister call phonebook lookup '("Maya Garcia","Chris Lynn")'
 ----
 +
-Because the program is written to return one value for one key, the command only returns information associated with the first key, in this example the phone number for `+Maya Garcia+`.
+Because the dapp is written to return one value for one key, the command only returns information associated with the first key, in this example the phone number for `+Maya Garcia+`.
 
 include::example$candid-ui.adoc[]
 +
 image:candid-phonebook.png[Phonebook functions]
 
-== Revise the source code in your program
+== Revise the source code in your dapp
 
 To extend what you have learned in this tutorial, you might want to try modifying the source code to return different results.
 
-For example, you might want to change the source code so that instead of a program that inserts and looks up a current key-value (name-phone) pair to create a program that stores contact information similar to a database "record" in which a primary key is associated with multiple fields.
-In this example, your program might enable users or another program to add information—such as a home phone number, a cell phone number, an email address, and a street address—and selectively return all or specific field values.
+For example, you might want to change the source code so that instead of a dapp that inserts and looks up a current key-value (name-phone) pair to create a dapp that stores contact information similar to a database "record" in which a primary key is associated with multiple fields.
+In this example, your dapp might enable users or another dapp to add information—such as a home phone number, a cell phone number, an email address, and a street address—and selectively return all or specific field values.
 
-== Stop the local network
+== Stop the local canister execution environment
 
-After you finish experimenting with your program, you can stop the local Internet Computer network so that it doesn’t continue running in the background.
+After you finish experimenting with your dapp, you can stop the local execution environment so that it does not continue running in the background.
```

Comment:
```suggestion
After you finish experimenting with your dapp, you can stop the local canister execution environment so that it does not continue running in the background.
```